source/release-notes.txt

```diff
@@ -10,6 +10,34 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.40.3
+----------------
+
+*Released October 11, 2023*
+
+New Features:
+
+- After creating a $search index you are now redirected to the search 
+  indexes modal (:issue:`COMPASS-7247`).
+- Field names are now autocompleted when defining indexes. (:issue:`COMPASS-7174`).
+- Syntax errors are now highlighted when defining indexes. (:issue:`COMPASS-7246`).
```

Comment:
(question / suggestion)
I know some of the Atlas Search features have been dropped into Release Notes before, but a quick scan didn't show any place where we said specifically "Hey! You can now create and manage Atlas Search indexes in Compass!" Is that true, and if so, perhaps we do that here? 

What about a slight rewrite to highlight the overall improvement being introduced, which I believe is significant. Something like 
- Introduced the ability to create and manage Atlas Search indexes in Compass. 
      - smaller feature ...
      - smaller feature ...
      - smaller feature ...
      - For more details, see Managing Indexes and Create an Atlas Search Index.

CC: @jaoppenheim for feedback